bridge/engine/EngineMixer.cpp

```diff
@@ -3646,18 +3646,14 @@ void EngineMixer::processBarbellMissingPackets(bridge::SsrcInboundContext& ssrcI
     }
 }
 
-void EngineMixer::processRecordingMissingPackets(const uint64_t timestamp)
+void EngineMixer::processRecordingUnackedPackets(const uint64_t timestamp)
```

Comment:
What this method actually does is to resend cached packets if they are not acked yet.
The shouldProcess is data race when called from here but we have not seen it in tcheck yet because we do not test recording.